case-studies/simple/draft1.md

```diff
@@ -0,0 +1,215 @@
+
+"There’s a long line of patients waiting outside their cabin," Deepa said, painting the scene. "They’re measuring a patient’s weight, BP, speaking to them, asking them about their day, their symptoms, whether or not they’ve been taking their meds, and recording all of this info in a register. Some of that information needs to go into _yet another_ register. In the meantime another patient who had been to see the doctor returns: the nurse quickly updates their info. Then they’re called away to attend to something for 10 minutes. It’s non-stop. And through all of this they’re polite, yet firm, and are making no mistakes. It's pretty amazing."
+
+No, the nurses could not wait for an app. The Simple app's nearest competitor was a piece of paper: If the nurse could write down a patient's results faster than they could enter it into Simple, the app had failed them. Not only could the app never wait for the network, it could absolutely never crash, hang, or pause --- not even for a second.
+
+Govind sat down to speak to a nurse during a break set aside to test the app. He was a bit surprised to see her pull a new iPhone out of her pocket and set it on the desk. Android has 95% of India's mobile marketshare. iOS has less than 4%. Statistically speaking, building Simple for Android felt like more of an inevitability than a design choice.
+
+She took a sample Android phone the team brought with them and began to examine Govind as she would treat any other patient. She wrapped the cuff of the blood pressure monitor around his arm and inflated it. As she opened the app to enter his BP reading, she exclaimed, "Oh! This is in Punjabi. That's not what I was expecting. Can you change the interface to English for me?"
+
+TODO: Hindi? (Govind)
+
+Govind switched the phone's locale to English and handed it back to her so she could finish trying the app while the designers watched and took notes. As she put the blood pressure monitor away, she turned to them. "You know, it is very nice of you all to build this app... I'm just not sure I trust it yet. I will use it because it is mandated but I will keep entering my patients' data into the paper register as well. These records are very important. It is hard for me to entrust my patient's lives to software that didn't even exist six months ago."
+
+TODO: Hindi? (Govind)
+
+The lessons in the other facilities were similar. The nurses were polite but cautious of Simple's promises. They were unflagging as a constant stream of patients poured through the clinic. And they remained full of surprises.
+
+"An robh thu ann an Alba? Tha e brèagha." ("Have you been to Scotland? It's beautiful.") one nurse asked with a calm smile as Srihari and Govind quizzed her about her time in Scotland, where she took her nursing degree.
```

Comment:
Wait the nurse actually spoke in Gaelic? I had no idea.

Reply:
"Based on a true story." :wink:

I'm happy to take this out if it's too far from the truth. I just thought it was cute.